### PR TITLE
Migrate to Gatsby

### PR DESCRIPTION
**Pull request by adamazad**
Now we need a build pipeline on this branch

### CODE REVIEW

**Comment by adamazad** [comment]
[f](https://unruffled-leakey-634230.netlify.app/)

**Comment by adamazad** [comment]
After a second thought, I think we want to replace _loans_ with _attacks_. The flash in the background represents flash loans. So, the whole logo reads as Prevent Flash Loan Attacks

**Comment by PatrickAlphaC** [comment]
@adamazad thanks for this PR! Agree. The logo looks like it says "prevent loans" at the moment. There is also no information in the readme about contributing, testing, or building. Why was that part taken out? I am not familiar with gatsby.

**Comment by adamazad** [comment]
The testing portion was taken out because it does not relate to Gatsby. It was generated by Jekyll. I will add the Contribution section back. It was a mistake.

**Comment by adamazad** [comment]
@PatrickAlphaC better? I will submit another PR to add CONTRIBUTING.md

image

**Comment by PatrickAlphaC** [comment]
Much better :) i still don't know how to build and deploy this though. If we are going to move from jekyll -> gatsby we'll need to know how to run it. 

I'm assuming this is it: https://www.gatsbyjs.com/docs/how-gatsby-works-with-github-pages/

But would like to see your readme notes so everyone can test with it.

**Comment by adamazad** [comment]
Patrick, please review the 3 new commits.

Yes, the link you provided is correct.

**Comment by PatrickAlphaC** [comment]
Nice! Looks like I was able to run a build!
We'll have to add something like this the github pages knows how to read from it 
```
Copypackage.json: copy code to clipboard
{
  "scripts": {
    "deploy": "gatsby build --prefix-paths && gh-pages -d public"
  }
}
```

So I think we just need 1 additional tweak for this PR. It looks like the background was removed here. Is there a nice default gatsby background we could use?

**Comment by adamazad** [comment]
@PatrickAlphaC you will need to:

1. Merge this PR to `main`
2. Clone the repo or update your local `main` branch to match.
3. Install packages: `yarn install`
4. Run `yarn deploy`. You have permission to deploy to the `gh-pages` branch.

Then, the site will be live. I do not have permission to setup a workflow. It requires adding a public key pair to the repo's Secrets.

**Comment by PatrickAlphaC** [comment]
Going to be reviewing and pulling in today!

Thanks again for this! I think we will use the background created from here: https://github.com/PreventFlashLoanAttacks/Prevent-Flash-Loan-Attacks/issues/3

**Comment by PatrickAlphaC** [comment]
Hi @adamazad, I wasn't able to pull this in yet. I'm having issues deploying it to github pages. Have you been able to deploy it to github pages before?

**Comment by adamazad** [comment]
Yes, I used `deploy` to publish it here https://github.com/adamazad/Prevent-Flash-Loan-Attacks/tree/gh-pages

**Comment by federicosan** [comment]
@PatrickAlphaC could you please tell me the size of the banner? so I can submit the PR with the correct sizes, If you have a deployed version of the site I can also adapt the background for mobile.

**Comment by PatrickAlphaC** [comment]
@adamazad hmm... let me try again. It wasn't working for me.

@federicosan I'm not sure. How big is your banner supposed to be?
